### PR TITLE
[inner-2351] print thread stackTrace

### PR DESCRIPTION
**Pull request by wenyh1**
Reason:  
  BUG #. or Improve #.  
Type:  
  BUG/Improve  
Influences：  
   fix xx

### CODE REVIEW

**Review by dcy10000 — DISMISSED** [review]
ok

File: src/main/java/com/actiontech/dble/util/ExecutorUtil.java [hunk 1]
```diff
@@ -66,7 +66,7 @@ private static NameableExecutor createCached(String name, int size, boolean isDa
      */
     public static NameableExecutor createCached(String name, int size, int maxSize, ThreadChecker checker) {
```
Comment (dcy10000) [inline src/main/java/com/actiontech/dble/util/ExecutorUtil.java:67]: this api is only made for timeExecuter. 
should use a special function name to distinguish.

**Review by dcy10000 — CHANGES_REQUESTED** [review]

**Review by dcy10000 — APPROVED** [review]
ok

File: src/main/java/com/actiontech/dble/util/ExecutorUtil.java (hunk 1 above)
Comment (wenyh1) [reply src/main/java/com/actiontech/dble/util/ExecutorUtil.java]: get

**Pull request merged**